posts/specfile-advanced/index.md

```diff
@@ -0,0 +1,252 @@
+---
+title: Advanced usage of the specfile library
+date: 2025-01-09
+authors: nforro
+tags:
+  - specfile
+---
+
+# Advanced usage of the specfile library
+
+Are you already familiar with the basic concepts of the _specfile_ library?
+Then welcome to a collection of some slightly more advanced topics that you could find useful.
+Otherwise you can take a look at [Introduction to specfile library](posts/specfile-introduction).
+
+<!--truncate-->
+
+## Forced parsing
+
+Imagine you want to make changes to a spec file that requires some sources
+to be present at the time of parsing, such as sources referenced from shell
+expansions, e.g. `%(date +%F -r %{SOURCE0})`, or includes some external files
+using the `%include` directive, or loads external macrofiles with the `%load` macro,
```

Comment:
```suggestion
expansions, e.g. `%(date +%F -r %{SOURCE0})`, includes some external files
using the `%include` directive, or loads external macrofiles with the `%load` macro,
```

Not sure about that _e.g._ with all the _or_s

Reply:
> Not sure about that _e.g._ with all the _or_s

How else would you write it? I wanted to give an example because _sources referenced from shell
expansions_ is a bit vague for most people, I think.

Reply:
Or do you think it's ok now with the removed or? :slightly_smiling_face:

Reply:
It's just because of the commas and an example among the enumeration of the possibilities of what user can do. :smile:

Reply:
Maybe this would be better?

```md
Imagine you want to make changes to a spec file that requires some sources
to be present at the time of parsing such as sources referenced from shell
expansions (e.g. `%(date +%F -r %{SOURCE0})`), includes some external files
using the `%include` directive, or loads external macrofiles with the `%load` macro,
but all you have is the spec file itself.
```

Reply:
Definitely <3